website/content/api-docs/coordinate.mdx

```diff
@@ -76,7 +76,8 @@ within the same area.
 ## Read LAN Coordinates for all nodes
 
 This endpoint returns the LAN network coordinates for all nodes in a given
-datacenter.
+datacenter and
+[whether some results were filtered out due to ACL policies](/api-docs#results-filtered-by-acls).
```

Comment:
@trujillo-adam : I agree that this isn't clear. Two approaches come to mind for how to address this. Let me know whether we should do 1, 2, or both:

1. Reword the link part as. "This endpoint also indicates via the [`X-Consul-Results-Filtered-By-ACLs` HTTP response header](/api-docs#results-filtered-by-acls) whether some results were filtered out due to ACL policies."
2. Add an example to https://www.consul.io/api-docs#results-filtered-by-acls showing how to make a request and find this header in a response using cURL.

Reply:
Or whether these seem in sufficient / there's a better option you think of.

Reply:
OOOHHH, so the response header could include the `X-Consul-Results-Filtered-By-ACLs` field. Is it a Boolean, like, `Content:X-Consul-Results-Filtered-By-ACLs: true` or just some unassigned string of metadata? I see a few other header fields sprinkled throughout the page, but I don't see like a reference for the response headers that our endpoints could return. To me, documenting that would be the priority because it could provide a scannable single point of reference. 
This is how I would reword that bit we're discussing:
"This endpoint returns the LAN network coordinates for all node in a given datacenter. If ACLs are enabled, the response header may include the `Content:X-Consul-Results-Filtered-By-ACLs: true` parameter, which indicates that the results were filtered according the ACL policy configuration. Refer to the API header reference for additional information."

Reply:
Regarding the suggestion to have a Consul HTTP API header reference page/section, [I had the same realization while working on this task](https://hashicorp.slack.com/archives/CPEPBFDEJ/p1646261798331069). I intend to create a Github issue for this so it can be worked on at some point. However, I don't think creating that reference (or a starting point for it) is something I have time for right now, so I was planning to keep that out of scope for this PR. Do you agree with that approach (addressing that header reference separately)?

#### Summary

Though I prefer option 1, I'm happy to go with either of the below options. Let me know and I'll proceed. I can define this in a partial so that we can reference the same text everywhere (instead of lots of copy and paste).

##### Option 1
> This endpoint returns the LAN network coordinates for all nodes in a given datacenter.
>
> The [HTTP response header `X-Consul-Results-Filtered-By-ACLs: true`](https://github.com/api-docs#results-filtered-by-acls) will be present if some results were filtered out due to ACL policy configuration.

##### Option 2
> This endpoint returns the LAN network coordinates for all nodes in a given datacenter.
>
> If ACLs are enabled, the HTTP response may include the header `X-Consul-Results-Filtered-By-ACLs: true` indicating that the results were filtered according the ACL policy configuration. Refer to the [`X-Consul-Results-Filtered-By-ACLs` header reference](https://github.com/api-docs#results-filtered-by-acls) for additional information.

#### Detailed Thoughts

**The thoughts below led to the summary above, you can probably skip this if short for time**

> If ACLs are enabled, the response header may include the X-Consul-Results-Filtered-By-ACLs: true parameter, which indicates that the results were filtered according the ACL policy configuration. Refer to the API header reference for additional information.

In my initial wording (`[whether some results were filtered out due to ACL policies](/api-docs#results-filtered-by-acls)`), I tried to keep the language on the endpoint itself concise and use the link to give more info about how to actually use it.  It sounds like as a reader yourself (when reviewing), the wording (though concise) wasn't communicating effectively. I was worried that writing 1-2 standalone sentences would be too much, and also be fully redundant with the link that provides "additional information" (that is only 2 sentences itself): https://www.consul.io/api-docs#results-filtered-by-acls

In an attempt to write something clearer that isn't fully redundant with the "additional info" link, I came up with the below... But the main reason it's shorter than what you suggested is that the link is inline rather than in a "Refer to ______ for additional info".

> This endpoint returns the LAN network coordinates for all nodes in a given datacenter
>
> This endpoint also indicates via the [X-Consul-Results-Filtered-By-ACLs HTTP response header](https://github.com/api-docs#results-filtered-by-acls) whether some results were filtered out due to ACL policies.

Reply:
It takes more space, but I prefer option 2 because it can be more helpful to give readers a sense of the topic you're sending them to. The issue with the style in option 1 (in which we link a phrase or keyword) is that the context of the reference may not match the context of linked phrase/keyword. It's not always the case, but when the linked phrase isn't necessarily the core message of the linked content, then it might be jarring for readers. 

Linked phrases/keywords may also affect the UX. In this instance, we've already provided the important information--the link is supplementary and may actually distract readers from accomplishing their current task.  

But we could be overthinking it.

Reply:
The text now says:
> The HTTP response header `X-Consul-Results-Filtered-By-ACLs: true` will be present if some results were filtered out due to ACL policy configuration. Refer to the [HTTP API documentation](https://consul-xuisg6nod-hashicorp.vercel.app/api-docs#results-filtered-by-acls) for more information.